vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,81 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const VUSDT = "0xfD5840Cd36d94D7229439859C0112a4185BC0255";
+export const VUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const VDAI = "0x334b3eCB4DCa3593BCCC3c7EBD1A1C1d1780FBF1";
+export const VFDUSD = "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba";
+export const VTUSD = "0xBf762cd5991cA1DCdDaC9ae5C638F5B5Dc3Bee6E";
+export const VUSDT_STABLE_COIN = "0x5e3072305F9caE1c7A82F6Fe9E38811c74922c3B";
+export const VUSDT_GAMEFI = "0x4978591f17670A846137d9d613e333C38dc68A37";
+export const VUSDT_DEFI = "0x1D8bBDE12B6b34140604E18e9f9c6e14deC16854";
+
+export const NEW_VUSDT_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
+export const NEW_VUSDC_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
+export const NEW_VDAI_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
+export const NEW_VFDUSD_IR = "0xE19C14171C2aC6CA63E008133e4B0D8571164BA3";
+export const NEW_VTUSD_IR = "0x2ba0F45f7368d2A56d0c9e5a29af363987BE1d02";
+
+export const NEW_VUSDT_STABLE_COIN_IR = "0x2ba0F45f7368d2A56d0c9e5a29af363987BE1d02";
+export const NEW_VUSDT_GAMEFI_IR = "0x5ECa0FBBc5e7bf49dbFb1953a92784F8e4248eF6";
+export const NEW_VUSDT_DEFI_IR = "0x5ECa0FBBc5e7bf49dbFb1953a92784F8e4248eF6";
+
+const vip304 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-304 Chaos labs recommendations (May 16th, 2024)",
+    description: ``,
```

Comment:
Add a description. VIP-289 can be used as a template

Reply:
Done https://github.com/VenusProtocol/vips/pull/290/commits/5e7e84853cbbd1889b42384f5110f6670a8b8337